bundles/org.openhab.binding.velbus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -642,6 +880,84 @@
 		<state readOnly="true" pattern="%.1f %unit%">
 		</state>
 	</channel-type>
+	<channel-type id="hour">
+		<item-type>Number</item-type>
+		<label>Hour</label>
+		<state readOnly="false" min="0" max="23">
+		</state>
+	</channel-type>
+	<channel-type id="minute">
```

Comment:
Can you combine the `minute` and `hour` channel in one quantity type channel `Number:Time`?

Reply:
These channels are used to set an "alarm clock", but the modules only support setting the hour and the minutes. Setting of the date and seconds components is not supported. If the minute and hour channels are combined in a Number:Time channel, won't it be strange that the end-user could provide a date and a seconds component, that would be ignored by the Velbus modules?